### DISCUSSION TITLE
Add the  AlignDETR algorithm to the mmdet. 给 mmdet 支持 AlignDETR  算法

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#658] [MMDetection]
# Add the  AlignDETR algorithm to the mmdet. 
 给 mmdet 支持 AlignDETR  算法 

Support training and inference of the AlignDETR algorithm in mmdet, with reference to the open source code  [AlignDETR]([https://github.com/FelixCaae/AlignDETR](https://github.com/FelixCaae/AlignDETR) [)]()) 

参考官方开源代码 [AlignDETR]([https://github.com/FelixCaae/AlignDETR](https://github.com/FelixCaae/AlignDETR) [)]()) ，在 mmdet 中实现支持该算法训练和推理

## Task Information

**Repository**：MMDetection

**Difficulty**：Hard

**Points**：80

**Estimated Time Consumption**：35 days

## Task Status

**Claimant's Github ID**：jiongjiongli

**Claim Time**：2023年08月15日

**PR Link**：https://github.com/open-mmlab/mmdetection/pull/10843

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by jiongjiongli** [comment]
Completed. PR: https://github.com/open-mmlab/mmdetection/pull/10843
